.github/workflows/playwright.yml

```diff
@@ -47,47 +30,13 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-node-
 
-      - name: install dependencies with Playwright browsers
-        env:
-          COMMIT_INFO_BRANCH: ${{ fromJson(steps.get_pull.outputs.data).head.label }}
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Install dependencies
         run: |
           npm ci
-          npx playwright install --with-deps
           npx playwright --version
 
-      - name: Playwright React tests
-        run: npm run test:ct
-
-      - uses: actions/upload-artifact@v3
-        if: always()
-        with:
-          name: playwright-report
-          path: playwright-report/
-          retention-days: 3
```

Comment:
**note**: this is an intentional change. Since we aren't using playwright's in-built html reporter, there will be no artifacts to upload